addons/rose/addon/components/rose/dropdown/button/index.hbs

```diff
@@ -12,9 +12,9 @@
     {{if @iconOnly "has-icon-only"}}'
 >
   <span class='rose-dropdown-button-wrapper'>
-    <span class='rose-dropdown-button-body'>
```

Comment:
I am fairly confident this class wasn't being used and confirmed there was no change.